content/en/home/installation.md

```diff
@@ -101,3 +101,13 @@ sudo zypper install crow-translate
 ```bash
 sudo eopkg it crow-translate
 ```
+
+#### Ubuntu, Linux Mint, KDE Neon, etc
+
+:package: [Personal Package Archive Ubuntu](https://launchpad.net/~jonmagon/+archive/ubuntu/crow-translate)
```

Comment:
Maybe just use `PPA` in all languages? Sounds more familiar to me. What do you think?
And maybe put this right after Arch and Manjaro? These are very popular distributions.
```suggestion
:package: [PPA](https://launchpad.net/~jonmagon/+archive/ubuntu/crow-translate)
```

Reply:
Yeah, good idea. I want to mention PPAs are hosted in launchpad, doesn't `Launchpad PPA` sound better?

Reply:
Yes, I agree :)